apps/storybook/src/MatrixVis.stories.tsx

```diff
@@ -12,41 +11,61 @@ const complexDataArray = getMockDataArray<H5WebComplex>(
   '/nD_datasets/twoD_cplx'
 );
 
-const Template: Story<MatrixVisProps> = (args) => <MatrixVis {...args} />;
-
 const formatMatrixValue = format('.3e');
 const formatMatrixComplex = createComplexFormatter('.2e', true);
 
-export const Default = Template.bind({});
-Default.args = {
-  dataArray,
-  formatter: (val) => formatMatrixValue(val as number),
-};
-
-export const Complex = Template.bind({});
-Complex.args = {
-  dataArray: complexDataArray,
-  formatter: (val) => formatMatrixComplex(val as H5WebComplex),
-  cellWidth: 232,
-};
-
-export const TypedArray = Template.bind({});
-TypedArray.args = {
-  dataArray: toTypedNdArray(dataArray, Float32Array),
-  formatter: (val) => formatMatrixValue(val as number),
-};
-
-export const StaticIndexCells = Template.bind({});
-StaticIndexCells.args = {
-  dataArray,
-  sticky: false,
-  formatter: (val) => formatMatrixValue(val as number),
-};
-
-export default {
+const meta = {
   title: 'Visualizations/MatrixVis',
-  parameters: { layout: 'fullscreen' },
-  decorators: [FillHeight],
   component: MatrixVis,
-  args: { cellWidth: 120 },
-} as Meta;
+  decorators: [FillHeight],
+  parameters: { layout: 'fullscreen' },
+  args: {
+    cellWidth: 120,
+  },
+} satisfies Meta<typeof MatrixVis>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default = {
+  args: {
+    dataArray,
+    formatter: (val) => formatMatrixValue(val as number),
+  },
+} satisfies Story;
+
+export const CellWidth = {
```

Comment:
New story to document `cellWidth` on its own (previously via the `Complex` story).